tests/common/block_utils.rs

```diff
@@ -0,0 +1,247 @@
+use cairo_vm::Felt252;
+use snos::execution::helper::ExecutionHelperWrapper;
+use snos::io::input::{ContractState, StarknetOsInput, StorageCommitment};
+use snos::io::InternalTransaction;
+use starknet_api::core::{ClassHash, CompiledClassHash, ContractAddress};
+use starknet_api::deprecated_contract_class::ContractClass as DeprecatedContractClass;
+use starknet_api::hash::StarkHash;
+use starknet_crypto::FieldElement;
+
+use crate::common::transaction_utils::to_felt252;
+
+pub fn deprecated_compiled_class(class_hash: ClassHash) -> DeprecatedContractClass {
+    let variants = vec![
+        AccountWithLongValidate(CairoVersion::Cairo0),
+        AccountWithoutValidations(CairoVersion::Cairo0),
+        ERC20,
+        Empty(CairoVersion::Cairo0),
+        FaultyAccount(CairoVersion::Cairo0),
+        // LegacyTestContract,
+        SecurityTests,
+        TestContract(CairoVersion::Cairo0),
+    ];
+
+    for c in variants {
+        if ClassHash(override_class_hash(&c)) == class_hash {
+            let result: Result<DeprecatedContractClass, serde_json::Error> =
+                serde_json::from_str(c.get_raw_class().as_str());
+            return result.unwrap();
+        }
+    }
+    panic!("No deprecated class found for hash: {:?}", class_hash);
+}
+
+pub fn compiled_class(class_hash: ClassHash) -> CasmContractClass {
+    let variants = vec![
+        AccountWithLongValidate(CairoVersion::Cairo1),
+        AccountWithoutValidations(CairoVersion::Cairo1),
+        Empty(CairoVersion::Cairo1),
+        FaultyAccount(CairoVersion::Cairo1),
+        TestContract(CairoVersion::Cairo1),
+    ];
+
+    for c in variants {
+        if c.get_class_hash() == class_hash {
+            let result: Result<CasmContractClass, serde_json::Error> = serde_json::from_str(c.get_raw_class().as_str());
+            return result.unwrap();
+        }
+    }
+    panic!("No class found for hash: {:?}", class_hash);
+}
+
+fn override_class_hash(contract: &FeatureContract) -> StarkHash {
+    match contract {
+        // FeatureContract::AccountWithLongValidate(_) => ACCOUNT_LONG_VALIDATE_BASE,
+        FeatureContract::AccountWithoutValidations(CairoVersion::Cairo0) => {
+            let fe = FieldElement::from_dec_str(
+                "3043522133089536593636086481152606703984151542874851197328605892177919922063",
+            )
+            .unwrap();
+            StarkHash::from(fe)
+        }
+        // FeatureContract::Empty(_) => EMPTY_CONTRACT_BASE,
+        FeatureContract::ERC20 => {
+            let fe = FieldElement::from_dec_str(
+                "2553874082637258309275750418379019378586603706497644242041372159420778949015",
+            )
+            .unwrap();
+            StarkHash::from(fe)
+        }
+        // FeatureContract::FaultyAccount(_) => FAULTY_ACCOUNT_BASE,
+        // FeatureContract::LegacyTestContract => LEGACY_CONTRACT_BASE,
+        // FeatureContract::SecurityTests => SECURITY_TEST_CONTRACT_BASE,
```

Comment:
Why are all these lines commented?

Reply:
Whole test setup is work in progress so I treat it as a draft leaving code commented out in case it might be useful later.